src/logger/environment.ts

```
@@ -0,0 +1,64 @@
+     * @returns Normalized NODE_ENV (trimmed and lowercase)
+     */
+    private static getNormalizedNodeEnv(): string {
+        return (process.env.NODE_ENV || '').trim().toLowerCase();
+    }
+
+    /**
+     * Determines the appropriate log mode based on NODE_ENV
+     * @returns LogMode appropriate for current environment
+     */
+    static getEnvironmentMode(): LogMode {
+        const nodeEnv = this.getNormalizedNodeEnv();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)